File: cfgs/pythoncfgs/NUMBER_DIGITS_PRODUCT_TWO_NUMBERS.py
Nodes
1;statement;foo_1 = 0;8;
2;statement;foo_1 = fun_1(foo_2 * foo_3);9;
3;conditional;foo_1 == 0;10;
4;conditional;foo_1 > 0;12;
5;statement;return foo_1;15;
6;statement;foo_1 = foo_2 + 1;13;
7;statement;foo_1 = foo_2 // 10;14;
8;statement;return 1;11;
9;exit;;
Edges
0;2;
1;3;
2;4;8;
3;5;6;
4;9;
5;7;
6;4;
7;9;
8;

File: cfgs/java_cfgs_foo/NUMBER_DIGITS_PRODUCT_TWO_NUMBERS.java
Nodes
1;special;;;
2;statement;int count = 0;14;
3;statement;int p = fun_1(foo_1 * foo_2);15;
4;conditional;foo_1 == 0;16;
5;statement;return 1;16;
6;exit;;;
7;conditional;foo_1 > 0;17;
8;statement;count++;18;
9;statement;p = foo_1 / 10;19;
10;statement;return foo_1;21;
Edges
0;2;
1;3;
2;4;
3;5;7;
4;6;
5;
6;10;8;
7;9;
8;7;
9;6;
